_posts/en/newsletters/2021-02-24-newsletter.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Bitcoin Optech Newsletter #137'
+permalink: /en/newsletters/2021/02/24/
+name: 2021-02-24-newsletter
+slug: 2021-02-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the results of discussion about
+choosing activation parameters for a taproot soft fork and includes our
+regular sections with selected questions and answers from the Bitcoin
+StackExchange, releases and release candidates, and notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+- **Taproot activation discussion:** Michael Folkson [summarized][folkson
+  lot] a second meeting about activation parameters for [taproot][topic
+  taproot], concluding that the meeting was "largely unproductive" about
+  choosing what value to use for the LockinOnTimeout (LOT) parameter of
+  [BIP8][].  However, there was near universal agreement for other
+  activation parameters, most notably reducing the amount of hash rate
+  that needs to signal for the fork to activate from 95% to 90%.
+
+    Discussion continued about the LOT parameter on the mailing list,
+    mainly about whether or not it should be exposed to users as an
```

Comment:
Exposing the choice to users was an option that was discussed on the mailing list but there wasn't much/any support for it. ZmnSCPxj effectively withdrew the suggestion. https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-February/018444.html

Reply:
From my perspective, nearly all of the discussion was in response to Samson Mow suggesting "release both [options]" and ZmnSCPxj suggesting a mandatory choice.  Here's the entire thread to date with Mow's post highlighted:

![2021-02-23-03_22_11_160666958](https://user-images.githubusercontent.com/61096/108849757-9f15b700-7586-11eb-89b6-bfc761d8d0e2.png)

Almost all of those responses are describing and evaluating the challenges of different people running with different LOT parameters.

Reply:
Right but the ZmnSCPxj suggestion of a mandatory user option within one software release was withdrawn and argued against. The suggestion that is still possible (likely even) is that the user chooses between running one software release with LOT=true set (e.g. a community release) and another with LOT=false set (e.g. a Core release).

Reply:
"whether or not it should be exposed to users as an option they can set (or are forced to set)."

This sounds to me like a user voluntarily or involuntarily choosing an option within a single release rather than a user choosing between two distinct releases to me. But certainly not asking for or requesting a change. Just attempting to help out in case you hadn't gone through the all the mailing list posts. It sounds like you have (in which case I am not offering anything here)

Reply:
I think I understand your concern now.  Thanks for being patient with me!

Reply:
@michaelfolkson edited this in 6c37e2efa74c11685f992d0d7e9bd6c849070d98 to hopefully address your concern.